target/target.go

```diff
@@ -77,100 +44,20 @@ func Glob(dst string, globs ...string) (bool, error) {
 // doesn't exist, it always returns true and nil.  It's an error if any of the
```

Comment:
Can you edit the second sentence of the comment so it says

If a source or destination is a directory, this function returns true if a source has any file that has been modified more recently than the most recently modified file in dst.

I think that would make the behavior more clear.

Reply:
Updated and reflowed